lib/src/bbob_parser/lexer.dart

```diff
@@ -157,32 +160,76 @@ class Lexer {
         }
       }
 
-      return !isEqual && !isWhiteSpace;
+      return !isWhiteSpace;
     }
 
-    void _nextAttr() {
-      final attrStr = attrCharGrabber.grabWhile(validAttr);
-      final currChar = attrCharGrabber.current;
+    // Represents: https://github.com/JiLiZART/BBob/blob/99c629e66678e40cec7b924eda2f1bfe0d46b941/packages/bbob-parser/src/lexer.ts#L104
+    TagState _nextTagState() {
+      if (tagMode == TagState.ATTRIBUTE) {
+         // Grab untill end, whitespace or end tag.
+         final validAttrName = (String? char) => !(char == '=' || _isWhiteSpace(char));
+         final name = attrCharGrabber.grabWhile(validAttrName);
+
+         final nextchar = attrCharGrabber.current;
+         final isEnd = attrCharGrabber.isLast;
+         final isValue = nextchar != equal;
+
+         attrCharGrabber.skip();
+
+         // Tag has ended already, this is a attribute value.
+         if (isEnd || isValue) {
+           final escaped = unquote(trimChar(name, doubleQuote));
+           attrTokens.add(Token(
+               TokenType.AttributeValue, escaped, _linePosition, _columnPosition));
+         } else {
+           // Defo a name
```

Comment:
AVOID abbreviations and acronyms unless they are obvious

https://dart.dev/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious

```suggestion
           // Definitely a name
```